[ENH] Make explicit that "task" metadata applies to "beh" modality (#804)
bids-standard/bids-specification: src/04-modality-specific-files/07-behavioral-experiments.md

`duration` columns can still be included, but should be labeled `_beh.tsv`
rather than `_events.tsv`.

Each task has a unique label that MUST only consist of letters and/or numbers
(other characters, including spaces and underscores, are not allowed) with the
[`task-<label>`](../99-appendices/09-entities.md#task) key/value pair.
Those labels MUST be consistent across subjects and sessions.

The OPTIONAL [`acq-<label>`](../99-appendices/09-entities.md#acq) key/value pair corresponds to a custom label to
distinguish different conditions present during multiple runs of the same task.
For example, if a study includes runs of an n-back task, with deep brain
stimulation turned on or off, the data files may be labelled
`sub-01_task-nback_acq-dbson_beh.tsv` and `sub-01_task-nback_acq-dbsoff_beh.tsv`.

> ## RECOMMENDED metadata

Reviewer: Maybe this should be made more clear that it addresses task data

```suggestion
## RECOMMENDED task metadata
```

In addition of the metadata REQUIRED for some data that can be found in the `beh` folder
(for example `SamplingFrequency` and `StartTime` for `*_<physio|stim>.tsv.gz` files),
it is RECOMMENDED to add the following metadata to the JSON files of this folder.

| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                      |
| ------------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |

> | TaskName           | RECOMMENDED           | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `TaskName` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. A RECOMMENDED convention is to name resting state task using labels beginning with `rest`. |

Reviewer 2: Rest doesn't really apply for `beh`.
```suggestion
| TaskName           | RECOMMENDED           | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `TaskName` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. |
```

| TaskDescription    | RECOMMENDED           | [string][]    | Description of the task.                                                                                                                                                                                                                                                                                                                                                             |
| Instructions       | RECOMMENDED           | [string][]    | Text of the instructions given to participants before the scan. This is not only important for behavioral or cognitive tasks but also in resting state paradigms (for example, to distinguish between eyes open and eyes closed).                                                                                                                                                    |

> | CogAtlasID         | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) term that describes the task (for example, Resting State with eyes closed "<https://www.cognitiveatlas.org/task/id/trm_54e69c642d89b>").                                                                                                                                                          |

Reviewer 2: ```suggestion
| CogAtlasID         | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) term that describes the task (for example, Stroop task "<https://www.cognitiveatlas.org/task/id/tsk_4a57abb949e27>").                                                                                                                                                          |
```

> | CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Rest "<http://wiki.cogpo.org/index.php?title=Rest>") .                                                                                                                                                                                                                     |

Author: ```suggestion
| CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Stroop "<http://wiki.cogpo.org/index.php?title=Stroop_Task_Paradigm>") .                                                                                                                                                                                                                     |
```

> | CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Rest "<http://wiki.cogpo.org/index.php?title=Rest>") .                                                                                                                                                                                                                     |

Reviewer 2: ```suggestion
| CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Rest "<http://wiki.cogpo.org/index.php?title=Stroop_Task_Paradigm>") .                                                                                                                                                                                                                     |
```

Author: just replacing your suggestion with one that changes rest to stroop in text **and** URL.

> | **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                      |
> | ------------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
> | TaskName           | RECOMMENDED           | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `TaskName` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. |
> | TaskDescription    | RECOMMENDED           | [string][]    | Description of the task.                                                                                                                                                                                                                                                                                                                                                             |
> | Instructions       | RECOMMENDED           | [string][]    | Text of the instructions given to participants before the scan. This is not only important for behavioral or cognitive tasks but also in resting state paradigms (for example, to distinguish between eyes open and eyes closed).                                                                                                                                                    |
> | CogAtlasID         | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) term that describes the task (for example, Stroop task "<https://www.cognitiveatlas.org/task/id/tsk_4a57abb949e27>").                                                                                                                                                          |
> | CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Stroop "<http://wiki.cogpo.org/index.php?title=Stroop_Task_Paradigm>") .                                                                                                                                                                                                                     |
> | InstitutionName    | RECOMMENDED           | [string][]    | The name of the institution in charge of the equipment that produced the composite instances.                                                                                                                                                                                                                                                                                        |
> | InstitutionAddress | RECOMMENDED           | [string][]    | The address of the institution in charge of the equipment that produced the composite instances.                                                                                                                                                                                                                                                                                     |

Reviewer 2: Mending fences.

```suggestion
| **Key name**       | **Requirement level** | **Data type** | **Description**
                                                               |
| ------------------ | --------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| TaskName           | RECOMMENDED           | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this `TaskName` field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. |
| TaskDescription    | RECOMMENDED           | [string][]    | Description of the task.                                                                                                                                                                                                                                                                  |
| Instructions       | RECOMMENDED           | [string][]    | Text of the instructions given to participants before the scan. This is not only important for behavioral or cognitive tasks but also in resting state paradigms (for example, to distinguish between eyes open and eyes closed).                                                         |
| CogAtlasID         | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [Cognitive Atlas](https://www.cognitiveatlas.org/) term that describes the task (for example, Stroop task "<https://www.cognitiveatlas.org/task/id/tsk_4a57abb949e27>").                                                                                  |
| CogPOID            | RECOMMENDED           | [string][]    | [URI][uri] of the corresponding [CogPO](http://www.cogpo.org/) term that describes the task (for example, Stroop "<http://wiki.cogpo.org/index.php?title=Stroop_Task_Paradigm>") .                                                                                                        |
| InstitutionName    | RECOMMENDED           | [string][]    | The name of the institution in charge of the equipment that produced the composite instances.                                                                                                                                                                                             |
| InstitutionAddress | RECOMMENDED           | [string][]    | The address of the institution in charge of the equipment that produced the composite instances.                                                                                                                                                                                          |
```
